source/contracts/LegacyReputationToken.sol

```diff
@@ -37,4 +37,19 @@ contract LegacyReputationToken is OldLegacyReputationToken {
     function onTokenTransfer(address, address, uint256) internal returns (bool) {
         return true;
     }
+
+    // Interface fulfilment
+    function trustedAuctionTransfer(address, address, uint256) public returns (bool) {
+        return true;
+    }
+
+    // Interface fulfilment
+    function mintForAuction(uint256) public returns (bool) {
+        return true;
+    }
+
+    // Interface fulfilment
+    function burnForAuction(uint256) public returns (bool) {
+        return true;
+    }
```

Comment:
We should not be adding any code to `LegacyReputationToken`.  We cannot change that code (it is already deployed) so changing it here is only lying to ourselves and likely to cause us to think something is working in test/test network only to find out it doesn't work on mainnet.

Reply:
This is the ReputationTokenV1 since in the context of v2 the LegacyReputationToken is the REP token of v1.

I guess I can just create a second IReputationToken interface instead of sharing the real one. Will do that.

Reply:
Oh actually this contract is not the _real_ Legacy REP token contract.

This is the one used in testing and development which includes a faucet function so it already is accepted to not be 1:1 with the real thing. _That_ contract is in the legacy_reputation folder as `OldLegacyReputationToken.sol`

Reply:
The test contract for LegacyReputationToken should not contain new REP interface.  I recognize the need for the faucet to make it work in test networks, but we don't want to accidentally code against the new token interface and then have it fail in production because the real thing doesn't have those methods.

Reply:
I'll do this because it's technically true, but in protest because the failure case would never happen for this code